pdns/recursordist/recursor-tsan.supp

```diff
@@ -1,3 +1,2 @@
 # We don't care about stats for now
-race:doStats
```

Comment:
I guess we don't need that anymore because we set `statistics-interval=0`? In that case I would prefer keeping the suppression though, as the suppression file might be used outside of the regression tests. I, for one, use it when I run with TSAN support enabled :)

Reply:
Will do, but in some case TSAN was not able to produce the correct backtrace, so the suppression of `doStats` did not work (as it is dependent on a proper unwinding of the stack).